webpack.config.babel.js

```diff
@@ -56,7 +65,7 @@ if (ENV === 'production') {
 
 const configDist = {
   context: `${__dirname}/src`,
-  entry: './index.js',
+  entry: ["babel-polyfill", './index.js'],
```

Comment:
where is the babel-polyfill being used? this might have something to do with the 26% increase in size of the sdk.